justfile

```diff
@@ -144,6 +144,9 @@ native-test:
 
 test: flutter-test native-test
 
+ln-dlc-node-test:
```

Comment:
perhaps a comment explaining why this one has its own recipe? e.g. `# slow tests, requiring `just docker` to be active`